Makefile

```diff
@@ -3,6 +3,7 @@
 IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 FLAGD_VERSION=v0.2.5
+CHART_VERSION=v0.2.6# x-release-please-version
```

Comment:
Make sure you include this `Makefile` in the list of files Release Please needs to manage.

https://github.com/open-feature/open-feature-operator/blob/main/release-please-config.json#L9

Reply:
@AlexsJones I did this.